test/functional/core/job_spec.lua

```diff
@@ -32,6 +32,9 @@ describe('jobs', function()
 
   before_each(function()
     clear()
+
+    assert:set_parameter('TableFormatLevel', 1000000)
```

Comment:
Ah didn't know about this. That's why I added this: https://github.com/neovim/neovim/commit/3d1ed7c959017dc8664497b26c86c9ffabf02891#diff-75e5cc780c1d61f0c237fc0c0055711bR431